generics/middleware/authenticator.js

```diff
@@ -47,7 +47,12 @@ module.exports = async function (req, res, next, token = '') {
 
 	// Allow search endpoints for non-logged in users.
 	let guestAccess = false
-	let guestAccessPaths = ['/dataPipeline/', '/templates/details', 'userProjects/certificateCallback']
+	let guestAccessPaths = [
```

Comment:
@Prajwal17Tunerlabs this has to be a secure URL must have internal access token,if not via header then via query param. It should not be a guest URL

Reply:
certificateCallback endpoint was already mentioned as a guest url hence i added certificateCalbackError endpoint, here i followed what vishnu told me. i tried to unlist these endpoint from guest urls and added internal-acces-token in the header just like you told me and ran the apis, but they wouldnt work